rero_mef/extensions/md5.py

```
@@ -0,0 +1,67 @@
+    Automatically computes and injects an ``md5`` field into every record before it is created or committed.  The hash
+    is calculated over the JSON-serialised record (sorted keys, ``$schema`` excluded) so that it can be used for cheap
+    change-detection during OAI harvesting.
+    """
+
+    def create_md5(self, record):
+        """Compute the MD5 hex-digest for *record*.
+
+        :param record: Record data (dict-like).
+        :returns: Hex-encoded MD5 string.
+        """
+        return hashlib.md5(
```

Ruff (0.15.10)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)